improve the help documents (#132)
kubernetes-sigs/kwok: docs/examples/kwokctl-snapshot.md

# Kwokctl Save/Restore a Cluster

> This document describes how to back up cluster data and restore data through `kwokctl`

Reviewer: ```suggestion
This document walks you through how to save and restore a cluster with `kwokctl`
```
